libs/etterlatte-ktor/src/main/kotlin/RouteUtils.kt

```diff
@@ -220,3 +222,15 @@ interface PersonTilgangsSjekk {
         bruker: Saksbehandler,
     ): Boolean
 }
+
+suspend fun PipelineContext<Unit, ApplicationCall>.hvisEnabled(
+    featureToggleService: FeatureToggleService,
+    toggle: FeatureToggle,
+    block: suspend PipelineContext<Unit, ApplicationCall>.() -> Unit,
+) {
+    if (featureToggleService.isEnabled(toggle, false)) {
+        block()
+    } else {
+        call.respond(HttpStatusCode.NotImplemented)
```

Comment:
```suggestion
        throw ForespoerselException(code = "NOT_IMPLEMENTED", status = 501, detail =  "Funksjonaliteten er ikke tilgjengelig enda.")
```
+ fjerne egen håndtering i frontend?

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/2895/commits/8a488274330342234e4a1da62ba1be93c1b129bf